tinygrad/renderer/triton.py

```diff
@@ -41,8 +41,10 @@ def define_scalar(local_size, dtype, args):
 
 def uops_to_triton(function_name:str, uops:List[UOp]):
   local_size: List[int] = []
+  dims: List[str] = []
+  valid: List[str] = []
   depth = 1
-  signatures, dims, bufs, kernel, valid = [], [], [], [], [] #type: ignore
+  signatures, bufs, kernel = [], [], []
```

Comment:
Would it be considered unnecessary to reduce these into one line separated by `;`? It would supposedly reduce the number of lines (https://discord.com/channels/1068976834382925865/1068991125353939066/1174459427559067770) and keep it readable, since these are just default, base initializations.

Reply:
semicolons shouldn't be used imo, its not a real line reduction, just code golfing.